data/transport missions.txt

```diff
@@ -3404,3 +3404,68 @@ mission "Capture Smuggler"
 		payment 150000
 		conversation
 			`The huge man drags the unconscious smuggler out of the cargo container with one hand and carries the cargo box with the other. He stops in front of you. "<first> <last>," he says. "My employer, he will be very happy. Your payment, you will receive soon. Is it not good, to be reliable?" The man leaves your ship, letting the smuggler's legs scrape along the ground as he walks. A moment later, you receive <payment> from an anonymous account.`
+
+
+
+mission "Inheritance Redirection"
+	description "Travel to <destination> to act as heir for the late warlord Limping Pappa."
+	source
+		government "Pirate"
+	destination
+		distance 7 9
+		government "Republic" "Free Worlds" "Syndicate"
+	to offer
+		credits >= 400000
+	on accept
+		payment -200000
+	on offer
+		conversation
+			`You are walking near the spaceport and suddenly a computer terminal starts blinking and beeping when you pass it.`
+			choice
+				`	(Check it out.)`
+				`	(Ignore it.)`
+					decline
+			`When you approach, you are greeted by a voice from the terminal: "Dear Sir or Madam, please respond!"`
+			choice
+				`	"Hello!"`
+				`	(Ignore it and walk away.)`
+					decline
+			`	"My name is Mr. Smith and I'm contacting you with a highly confidential and urgent proposal. I am, or more accurately was, the financial manager of the recently deceased warlord Limping Pappa. I am looking for a highly trustworthy individual who can help me relocate Mr. Pappa's fortune of 300 million credits."`
+			choice
+				`	"How did Mr. Pappa acquire his fortune exactly?"`
+				`	"What are you proposing?"`
+					goto proposal
+				`	"This sounds like a credits-stealing scam, I will not fall for this!"`
+					decline
+			`	"Oh, euuh, Mr. Pappa was specialized into tricking rich victims to transfer credits to him.`
+			label proposal
+			`	"I obviously cannot make myself a heir to Mr. Pappa. So what I propose is to present you as heir and have the inheritance transferred to you.`
```

Comment:
`a heir` → `an heir`. `An` is used before a word starting with a vowel, and since the `h` is silent, it's treated as though it's a word starting with a vowel. See for example [this Wikipedia article](https://en.wikipedia.org/wiki/Heir_apparent).

Also, I think that `you as heir` should be changed to `you as an heir`, to be consistent with the previous statement.

Reply:
Fixed as suggested.